backend/src/common/utility/validation.utils.ts

```diff
@@ -0,0 +1,7 @@
+export class ValidataionUtils {
+    static isString(val: any | undefined | null) {
+        const isTruthly = Boolean(val);
```

Comment:
`const isTruthy = !!val` visai nieko construktas, bet tavo tikrai nera blogai.
Be to, type zodyje `Truthy`